src/mythos/AllEncounterCardsBag.ttslua

```diff
@@ -146,12 +175,12 @@ function onObjectSpawn(obj)
   if indexCount == 0 then return end
   if obj.hasTag("Replaced") then return end
 
-  if obj.type == "Card" then
-    handleSpawnedCard(obj)
-  elseif obj.type == "Deck" then
+  if obj.type == "Deck" then
```

Comment:
Would you mind keeping the `handleSpawnedCard` function? Since it uses a separate index and has some special handling, I think it would be better to keep it.